### PR TITLE
Add tutorial: "Install NixOS and keep configuration on Git"

### LINKED ISSUE #11: Add tutorials section
In contrast to **blog** (which may become out of date over time), add a **tutorial** section.

To begin with, add our first tutorial titled *Install NixOS and keep configuration on Git* which I believe will be useful to many newcomers to go from "New computer" to "NixOS configuration on GitHub".

### PR DESCRIPTION
**Pull request by srid**
Resolves #11 

cc @shivaraj-bh for feedback. Gonna publish tomorrow.

To preview locally, run:

```sh
nix --refresh run github:nixos-asia/website/tutorial-nixos-config#preview \
  -- -o /en/tutorial/nixos-config
```

### CODE REVIEW

**Comment by srid** [comment]
FTR, I tested these on a Parallels NixOS VM (aarch64-linux) on mac.

File: en/tutorial/nixos-config.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+
+# Install NixOS and keep configuration on Git
```
Comment (srid) [inline en/tutorial/nixos-config.md:2]: Wonder if there's a better title ...

**Comment by srid** [comment]
There will probably be a "Part 2" to this, as a separate tutorial. That will mention `nixos-flake` among other workflows.

File: en/tutorial/nixos-config.md (hunk 1 above)
Comment (shivaraj-bh) [reply en/tutorial/nixos-config.md:2]: How about "Git-Managed NixOS configuration"?

File: en/tutorial/nixos-config.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+```sh
+$ nix --extra-experimental-features 'nix-command flakes' flake show /etc/nixos
+warning: creating lock file '/etc/nixos/flake.lock'
+error:
+       … while updating the lock file of flake 'path:/etc/nixos?lastModified=1702156351&narHash=sha256-km4AQoP/ha066o7tALAzk4tV0HEE%2BNyd9SD%2BkxcoJDY%3D'
+
+       error: opening file '/etc/nixos/flake.lock': Permission denied
+```
+
+Alright, now Nix understably cannot to write to root-owned directory. At this point, we are better off moving the whole configuration to our home directory, which would also prepare the ground for storing it on Git.
```
Comment (shivaraj-bh) [inline en/tutorial/nixos-config.md:106]:
```suggestion
Alright, now Nix understandably cannot write to root-owned directory. At this point, we are better off moving the whole configuration to our home directory, which would also prepare the ground for storing it on Git.
```

**Review by shivaraj-bh — APPROVED** [review]

**Comment by srid** [comment]
Merging this now, but let's announce tomorrow. 

@shivaraj-bh Can you post this on r/nixos close to EST/PDT Monday morning (Monday evening/night IST time)?

**Pull request merged**